crates/y-sweet-worker/Cargo.toml

```diff
@@ -27,3 +27,6 @@ futures = "0.3.28"
 thiserror = "1.0.44"
 serde = { version = "1.0.177", features = ["derive"] }
 serde_json = "1.0.104"
+rusty-s3 = "0.4.1"
+reqwest = "0.11.18"
```

Comment:
huh neat, so this just uses reqwest's wasm support?

since reqwest can also work with tokio, should we use this `s3_store` implementation in the native client too? If `rusty-s3` feels good to you, I like the idea of replacing the other library we currently use.

Reply:
I agree with that, ill open another pr that does that.

Reply:
sounds good, I'm all for doing this one step at a time.